api/openapi-spec/swagger.json

```diff
@@ -4121,6 +4121,10 @@
       "description": "If specified will pass option 67 to interface's DHCP server\n+optional",
       "type": "string"
      },
+     "extraOptions": {
```

Comment:
@davidvossel , thank you for pointing out. I have committed changes after running make generate.